__test__/main.test.ts

```diff
@@ -1,31 +1,58 @@
 import * as core from "@actions/core";
-import * as im from "@actions/exec/lib/interfaces";
+import * as actions_exec from "@actions/exec";
 
 import * as linux from "../src/setup-ros-linux";
 import * as osx from "../src/setup-ros-osx";
 import * as windows from "../src/setup-ros-windows";
-import * as utils from "../src/utils";
-
-utils.lib.exec = jest.fn(async (commandLine: string,
-    args?: string[],
-    options?: im.ExecOptions,
-    log_message?: string) => 
-    {
-        const argsAsString = (args || []).join(" ");
-        const message = log_message || `Invoking "${commandLine} ${argsAsString}"`;
-        return core.group(message, async () => {
-            return 0;
-        });
-    });
-
-test('run Windows workflow', async () => {
-    return windows.runWindows();
-})
 
-test('run Linux workflow', async () => {
-    return linux.runLinux();
+
+describe('basic workflow tests', () => {
+    beforeAll(() => {
+        jest
+            .spyOn(actions_exec, 'exec')
+            .mockImplementation(jest.fn());

```

Comment:
Any reason to remove previous implementation? It is helpful to just log to console what is being executed when testing.

Reply:
The `utils.exec` will still do the logging to console. My previous pull request was [duplicating the logic from `utils.exec`](https://github.com/ros-tooling/setup-ros/blob/6bd377f9c0cafbd82c33475f2726df2c0631b9c8/__test__/main.test.ts#L14) and now since I know to mock the `actions_exec.exec` so I think we no longer need that.